student/pom.xml

```diff
@@ -47,7 +49,17 @@
 			<groupId>org.opentestsystem.delivery</groupId>
 			<artifactId>tds-dll-mysql</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.opentestsystem.exam</groupId>
+			<artifactId>tds-exam-client</artifactId>
+			<version>${tds-exam-client.version}</version>
+		</dependency>
 
+		<dependency>
+			<groupId>org.opentestsystem.common</groupId>
+			<artifactId>tds-common-legacy</artifactId>
+			<version>${tds-common-legacy.version}</version>
+		</dependency>
```

Comment:
The dependency I was referencing was the auto value dependency. This is needed because of the Response object in the common legacy project.